### ISSUE TITLE
整理数据集

### ISSUE THREAD

**Issue by lambda-zhang** [paulxiong/cervical#3]
## 整理好的数据集服务器上的路径在这里
```
$ ls /opt/lambda_workspace/datasets/ -l
total 4
drwxrwxr-x 4 atec atec  24 Jun  6 15:30 0606
-rw-rw-r-- 1 atec atec 982 Jun 11 15:35 README.md
```

### 原始图片是这个GPU服务器上
```
$ ls /opt/zhuoyao_workspace/medical_ai/datasets/origin_train/                                                                                                                                                
$ ls /opt/zhuoyao_workspace/medical_ai/datasets/origin_annotation/       
```

[生成方法参考是这个](https://github.com/paulxiong/cervical/wiki/%E4%B8%80%E6%AD%A5%E6%AD%A5%E9%AA%8C%E8%AF%81%E5%BD%93%E5%89%8D%E4%BB%A3%E7%A0%81%E4%BD%BF%E7%94%A8%E6%96%B9%E6%B3%95(2019-06-29)              
)

下面是文件夹的描述

| 文件夹  | 数据集描述                                                  |
|----|---------------------------------------------------------------------------------|
| a  | Margin 切割出来的,没做其他处理                                                  |
| a2 | 在a的基础上删除Norm, 原始数据医生只标注了阴性，阳性没标注所以认为阳性数据不准确 |
| a3 | 在a2的基础上删除认为不清晰的，留下认为清晰的                                    |
| a4 | 被删除的，认为是不清晰的                                                        |
|    |                                                                                 |
|    |                                                                                 |
| b  | Mask 切割出来的,没做其他处理                                                    |
| b2 | 在b的基础上删除Norm, 原始数据医生只标注了阴性，阳性没标注所以认为阳性数据不准确 |
| b3 | 在b2的基础上删除认为不清晰的，留下认为清晰的                                    |
| b4 | 被删除的，认为是不清晰的                                                        |                                                    |
                                                                          
a4和b4是丢弃的图片，看上去类似下面这个
![2019-06-11 20:08:13屏幕截图](https://user-images.githubusercontent.com/4313090/59270767-e827b980-8c84-11e9-86c4-bd79a11ef3b9.png)

### 用文件夹下面的generate_npy.py生成npy文件
```
创建如下目录结构的目录，把对应的图片拷贝到名字对应的目录，执行generate_npy.py就生成了npy
train_datasets/
├── HPV
├── HSIL
├── LSIL
├── Norm
└── SCC
```
